DailyCodingProblem/Program.cs

```diff
@@ -211,5 +213,32 @@ static int CheckDecimalValueRange(int value)
                 return value;
             }
         }
+
+        /// <summary>
+        /// Given the mapping a = 1, b = 2, ... z = 26, and an encoded message, count the number of ways it can be decoded.
+        /// For example, the message '111' would give 3, since it could be decoded as 'aaa', 'ka', and 'ak'.
+        /// You can assume that the messages are decodable. For example, '001' is not allowed.
+        /// </summary>
+        static int Problem7(string message)
+        {
+            int path = 0;
```

Comment:
1. This should come after your base case, as there is no point setting it if you are going to return
2. I think you could improve the variable name here as path seems like you are returning the entire path it took, but really its just a count of the number of paths